### PR TITLE
Release process

### PR DESCRIPTION
**Pull request by rebecca-caktus**
Added version 1.0.0 (in development), an initial changelog, and documentation about the release process.

### CODE REVIEW

**Comment by emullaney** [comment]
Getting a sphinx error on "make html" with the conf.py changes

File: docs/source/releases/deployment-process.rst [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+    fab staging deploy
+
+
+.. _reset-staging-environment:
+
+Reset staging environment
+-------------------------
+
+These steps will restore the production database aand code states to the
```
Comment (emullaney) [inline docs/source/releases/deployment-process.rst]: aand/and

**Comment by emullaney** [comment]
Minor spelling fix and :rocket:

**Comment by rebecca-caktus** [comment]
We resolved the sphinx error with `add2virtualenv /path/to/tracpro-repo/`.

**Pull request merged**